### PR TITLE
Translate into Chinese

### PR DESCRIPTION
**Pull request by node**

### CODE REVIEW

**Comment by semioticrobotic** [comment]
This is very cool to see! Let us know if we can answer any questions for you, @node.

**Comment by node** [comment]
Finished !  

Would we invite someone else to help to review this translation work ?

**Comment by semioticrobotic** [comment]
> Finished !

Wow! That's wonderful news, @node! Thanks so much for working on this.
 
> Would we invite someone else to help to review this translation work ?

Ideally, yes—we would. But at the moment I can't quite think of anyone who might serve as reviewer. @LauraHilliger, do you know anyone? Or maybe @Hleson?

**Comment by LauraHilliger** [comment]
I'm sharing around, let's see if we can find someone! Thanks @node, this is awesome!!

**Comment by irvin** [comment]
I haven't proofread the translation yet, but one suggestion is to move it into "Simplified Chinese" folder so that we may have a place for Traditional Chinese version in the future.

File: Translations/Chinese/open-organization-definition.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# 开放式组织定义
+
+## 前言
+在各种规模的群体和团队为实现共同目标而开展合作的方式中，开放正变得越来越重要。如今，最具前瞻性的组织——无论其使命如何——都将开放作为迈向成功的必要方向。他们已经看到，开放可以带来：
+
+- **更加敏捷**, 因为成员更有能力以共同愿景齐心协力实现目标；
+- **更快创新**, 因为来自组织内外的想法都能得到更公平的考虑和更快速的试验，以及；
+- **增强参与**, 因为成员们清楚地看到他们的具体活动与组织的总体价值观、使命和精神之间的联系。
+
+但开放是不稳定的，是多层面的，是会带来争议的。
```
Comment (wengkee) [inline Translations/Chinese/open-organization-definition.md]: 原文 “But openness is fluid”，表达的是流动性，或者是动态的。我认为 “不稳定 / not stable” 跟这里表达的意思不太一样。改为“但开放是流动性的”，会不会更好一些？

File: Translations/Chinese/open-organization-definition.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- 透明性 
+- 包容性 
+- 适应性
+- 协作性
+- 社区性
+
+## 开放式组织的特征
+开放式组织有多种形式，它们的规模、组成和使命各不相同，但下文的五个特征是所有开放式组织的共同标志。
+
+在实践中，每个开放式组织都可能或多或少地以不同的方式体现这些特征中的每一个。此外，一些不认为自己是开放式组织的组织仍然可能拥抱其中的一些特征。但真正开放的组织体现了所有这些特征，并且以强大而富有成效的方式将它们联系起来。
```
Comment (wengkee) [inline Translations/Chinese/open-organization-definition.md]: "一些不认为自己是开放式组织的组织仍然可能拥抱其中的一些特征" 的拥抱感觉不是很正确，原文为 embrace，翻译为采用或者接纳会不会比较好？

File: Translations/Chinese/open-organization-definition.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- 人们对于成功和失败都很重视，因为它们都能提供经验教训。
+- 目标公开且明确，项目工作人员有明确的角色和责任，以加强问责制。
+
+
+### 包容性
+开放式组织具有包容性。他们不仅欢迎不同的观点，而且还实施特定的机制，随时随地邀请多种观点进行对话。感兴趣的各方和新人可以开始为组织提供帮助，而无需寻求每个利益相关者的明确许可（另请参阅*协作性*）。参与规则和协议很明确（另见*透明性*），并根据经过审查的通用标准运作。
+
+在开放式组织中，包容性是这样的：
+
+- People work transparently and share materials via common standards and/or agreed-upon platforms that do not prevent others from accessing or modifying them.
```
Comment (wengkee) [inline Translations/Chinese/open-organization-definition.md]: this should be removed

**Comment by wengkee** [comment]
I have reviewed and put in my comments for _open-organization-definition.md_, the rest LGTM

**Comment by node** [comment]
I have revised according to the above review comments. 
Thanks for your comments.  😄  
@wengkee @irvin

**Comment by wengkee** [comment]
LGTM @semioticrobotic @LauraHilliger

**Comment by semioticrobotic** [comment]
Seeing successful reviews and revisions, I feel comfortable merging this translation and will do so. Thanks, @node, for this work. We really appreciate it! And additional thanks to @wengkee for the thorough, kind, and generous review.

**Pull request merged**